articles/explainable-ai/index.md

```diff
@@ -0,0 +1,164 @@
+
+#### Online Recommendations
+A key application of machine learning is making recommendations to users. We receive product recommendations every day. Traditional machine learning methods predict the next best recommendation. For businesses selling products, this does not guarantee conversions. A person may receive recommendations but still abandon their shopping cart. It would assist businesses to understand how the system is making these predictions.
+
+Businesses would then fine-tune a potential shopping experience. Explainable AI would predict what the customer wants and explain the “why” as well. This results in more personalized recommendations and overall experience for the users. The greater the personalization, the higher the chance of getting the recommendations right. This also provides an opportunity to make further recommendations. There is a good chance customer market size is going to increase as well. The volume of sales per customer and conversion rates will increase.
+
+#### Credit and Loan Decision Making
+A very widespread use of machine learning is determining who gets access to certain financial services. We trust these algorithms to make these decisions fairly. Although, if we are denied a loan, the reasoning behind the denial may not be provided.
+
+Most traditional machine learning approaches can be categorized as black-box approaches. They would only predict who should receive or be denied financial services without any explanation. The implementation of explainable AI would explain why someone should receive or be denied financial services. This offers transparency to the process. It also makes the process more auditable. Legal compliance is also improved since decisions can be justified within the scope of the applicable laws.
+
+### Explainable AI Techniques/ Approaches
+To develop explainable systems, two main techniques are used; *ante-hoc and post-hoc*.
+
+#### Ante-Hoc Methods
+These techniques entail involving explainability into a AI model from the very beginning.
+
+##### Reverse Time Attention Model (RETAIN)
+To assist doctors with understanding AI software predictions, the [RETAIN](https://arxiv.org/abs/1608.05745) model was developed. It utilizes two recurrent neural networks, each having an attention mechanism. The two RNNs received patient hospital visit data. The attention mechanism was responsible for explaining the focus of the neural network and how a choice was influenced.
```

Comment:
This section needs more information. How was the model used to assist the doctor? What was the problem the doctor had, and how was it solved by RETAIN?